webapp/crashstats/supersearch/tests/test_views.py

```diff
@@ -630,48 +469,44 @@ def mocked_supersearch_get(**params):
             ],
         )
 
-    def test_search_custom_parameters(self):
-        self.create_custom_query_perm()
-
-        def mocked_supersearch_get(**params):
-            assert "_return_query" in params
-            assert "signature" in params
-            assert params["signature"] == ["nsA"]
-
-            return {
-                "query": {"query": None},
-                "indices": ["socorro200000", "socorro200001"],
-            }
-
-        SuperSearchUnredacted.implementation().get.side_effect = mocked_supersearch_get
+    def test_search_custom_parameters(self, client, db, es_helper, user_helper):
+        es_helper.health_check()
+        user = user_helper.create_protected_plus_user()
+        client.force_login(user)
 
         url = reverse("supersearch:search_custom")
-        response = self.client.get(url, {"signature": "nsA"})
+        response = client.get(url, {"signature": "nsA"})
         assert response.status_code == 200
         assert "Run a search to get some results" in smart_str(response.content)
-        assert "{&#34;query&#34;: null}" in smart_str(response.content)
-        assert "socorro200000" in smart_str(response.content)
-        assert "socorro200001" in smart_str(response.content)
-
-    def test_search_query(self):
-        self.create_custom_query_perm()
-
-        def mocked_query_get(**params):
-            assert "query" in params
-
-            return {"hits": []}
-
-        Query.implementation().get.side_effect = mocked_query_get
+        # Assert the query is in the editor
+        assert "{&#34;query&#34;:" in smart_str(response.content)
+        # Assert the available indices are there
+        template = es_helper.get_index_template()
+        now = utc_now()
+        indices = [
+            (now - datetime.timedelta(days=7)).strftime(template),
+            now.strftime(template),
+        ]
+        assert ",".join(indices) in smart_str(response.content)
+
+    def test_search_query(self, client, db, es_helper, user_helper):
+        user = user_helper.create_protected_plus_user()
+        client.force_login(user)
 
         url = reverse("supersearch:search_query")
-        response = self.client.post(url, {"query": '{"query": {}}'})
```

Comment:
This is not a valid Elasticsearch query but since it was all mocked out, the test never failed.